sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_generated/_configuration.py

```diff
@@ -1,53 +1,61 @@
 # coding=utf-8
 # --------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License.txt in the project root for
-# license information.
-#
+# Licensed under the MIT License. See License.txt in the project root for license information.
 # Code generated by Microsoft (R) AutoRest Code Generator.
-# Changes may cause incorrect behavior and will be lost if the code is
-# regenerated.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
 # --------------------------------------------------------------------------
+
+from typing import Any
+
 from azure.core.configuration import Configuration
 from azure.core.pipeline import policies
 
-from ._version import VERSION
-
+VERSION = "unknown"
```

Comment:
This should be 3.0.0?

Reply:
with the new autorest, we only generate a version file off of package-version, so if package-version is not set, we use unknown and don't write a version file.

I can generated with package version of 3.0.0-preview.1, since it's not quite 3.0.0. This also ties into whether you want the user agent to include, because if we're going to overwrite the sdk moniker, there's not a big point in having a version in the generated code.

Reply:
I think we should just generate with `--package-version=3.0.0b4` and remove the user_agent.py